fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatistic.java

```diff
@@ -395,4 +413,33 @@ public static ColumnStatistic createUnknownByDataType(DataType dataType) {
                     .build();
         }
     }
+
+    /**
+     * Get the map of column value and its row count percentage in the table.
+     * The hotValue is like:
+     * value1:percent1;value2:percent2;value3:percent3
+     * We only return the column value whose percentage is larger than 30%.
+     * @return Map of value -> percentage.
+     */
+    public Map<String, Float> getHotValues() {
+        if (hotValue == null || hotValue.isEmpty()) {
```

Comment:
replace String key by literal key, please
Maybe we should cache this map for performance reason